package.json

```diff
@@ -1,82 +1,82 @@
 {
-  "name": "the-chat-app",
```

Comment:
If that's prettier then let's revert. Otherwise npm will reformat it back again.